site/en/docs/search_reranking_using_embeddings.ipynb

```diff
@@ -31,19 +31,6 @@
         "# limitations under the License."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import pprint
</pre><p> </p><p>unused?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>It turns out I prefer <code>json.dumps(obj, indent=4)</code> for compatible objects.</p><p> </p>